scripts/spack/repo.yaml

```
@@ -1,2 +1,2 @@
 repo:
-  namespace: serac
+  namespace: spack
```

unless you meant to do this

```suggestion
  namespace: smith
```